feature_tests/src/attrs.rs

```diff
@@ -230,7 +230,8 @@ pub mod ffi {
 
     impl StructWithAttrs {
         // Dart backend does not support failable constructors on struct types
-        #[diplomat::attr(all(not(dart), supports = fallible_constructors), constructor)]
+        #[diplomat::attr(dart, disable)]
```

Comment:
praise: yeah this is the better way of doing it. we should fix that dart issue if possible though, but that's not your problem

(cc @robertbastian )